support/nrsc5.py

```diff
@@ -352,9 +352,16 @@ def callback_closure(evt, opaque):
         self.callback_func = CFUNCTYPE(None, POINTER(_Event), c_void_p)(callback_closure)
         NRSC5.libnrsc5.nrsc5_set_callback(self.radio, self.callback_func, None)
 
-    def pipe_samples(self, samples):
+    def pipe_samples_cu8(self, samples):
         if len(samples) % 4 != 0:
             raise NRSC5Error("len(samples) must be a multiple of 4.")
-        result = NRSC5.libnrsc5.nrsc5_pipe_samples(self.radio, samples, len(samples))
+        result = NRSC5.libnrsc5.nrsc5_pipe_samples_cu8(self.radio, samples, len(samples))
+        if result != 0:
+            raise NRSC5Error("Failed to pipe samples.")
+
+    def pipe_samples_cs16(self, samples):
```

Comment:
This API is a little strange because in C samples is ```int16_t *``` but in Python it is a byte array.

Reply:
I went that route because the most likely use case is reading in samples from a file, which will arrive in a `bytes` object. I wanted to avoid having to unpack & repack the data. If you have a better idea, I'd be happy to reconsider it.

Reply:
I didn't have any immediate idea without relying on numpy, and I don't think it is so important that adding a dependency is worth it.